sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfigBuilder.java

```diff
@@ -11,7 +11,6 @@
 /**
  * The throughput control group config builder.
  */
-@Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public class ThroughputControlGroupConfigBuilder {
```

Comment:
There are multiple APIs in this class which have been deprecated for more than 5 months now. Should we also remove them ? Since we are GAing the other ones, it would be better to remove them. 
@xinlian12 thoughts?

Reply:
see above - let's deprecate after holiday season to not risk regression when shipping hotfixes